Gem/Swap/Scenes/SwapScene.swift

```diff
@@ -210,14 +223,21 @@ extension SwapScene {
 
     private func onChangeFromAsset(_: AssetData?, _: AssetData?) {
         model.resetValues()
+        model.setSelectedSwapQuote(nil)
         focusedField = .from
         fetch()
     }
 
     private func onChangeToAsset(_: AssetData?, _: AssetData?) {
         model.resetToValue()
+        model.setSelectedSwapQuote(nil)
         fetch()
     }
+    
+    private func onChangeSelectedQuote(_: SwapQuote?, quote: SwapQuote?) {
+        guard let quote, let toAsset else { return }
+        model.updateToValue(for: quote, asset: toAsset.asset)
```

Comment:
```suggestion
        model.onSelectQuote(for: quote, asset: toAsset.asset)
```

this is a better naming for what it does